lib/country-iso-3166-1-alpha-3-codes.ts

```diff
@@ -0,0 +1,195 @@
+export const countryNameToIso31661Alpha3CodeMap: Record<string, string> = {
```

Comment:
I think I'm probably going to start saving the three letter country codes in the backend at some point so that the names in Airtable aren't coupled to the react app but this will have to do for now.

Basically know that things better than this will be coming at some point so I wouldn't get too concerned about this one :)